xray/utils/resultstable.go

```diff
@@ -152,6 +156,9 @@ func prepareViolations(violations []services.Violation, multipleRoots, isTable,
 	sort.Slice(securityViolationsRows, func(i, j int) bool {
 		if securityViolationsRows[i].SeverityNumValue != securityViolationsRows[j].SeverityNumValue {
 			return securityViolationsRows[i].SeverityNumValue > securityViolationsRows[j].SeverityNumValue
+		} else if securityViolationsRows[i].Applicable != securityViolationsRows[j].Applicable {
+			return getApplicableCveNumValue(securityViolationsRows[i].Applicable) >
+				getApplicableCveNumValue(securityViolationsRows[j].Applicable)
```

Comment:
Let's check if we can reuse this sorting code with the sorting code in line 235.